### ISSUE TITLE
Loss function modification feasibility

### ISSUE THREAD

**Issue by zzhan127** [junyanz/pytorch-CycleGAN-and-pix2pix#1265]
Hello,

I have a quick question about modifying the loss functions for pix2pix model. My work is related to extracting features, such as median, kurtosis, or some texture matrices, from deep learning predicted images. To make these features closer to ground truth images, is it feasible to use the features differences (kurtosis, median, texture matrices differences) directly as the loss function of pix2pix model?

Thanks!
